tests/EdgeToEdge/Routes/ValidateAddressRouteTest.php

```diff
@@ -42,7 +42,7 @@ public function testGivenInvalidCompanyAddress_validationReturnsErrorMessage():
 		$expectedResponse = [
 			'status' => 'ERR',
 			'messages' => [
-				'company' => 'field_required'
+				'companyName' => 'missing'
```

Comment:
I do not know if this change is fine. I tried finding code using this data but was unable to. Is it used somewhere in the UI?

Reply:
This is fine & fixes a potential problem. "Potential" because the JS in the forms does not call "validate-address" before all fields are filled, so we never got the wrong field name back. This is why we never caught this.